lib/ua_parser/storage.ex

```diff
@@ -1,46 +1,16 @@
 defmodule UAParser.Storage do
   @moduledoc """
-  Storage of User-Agent regular expressions.
+  Compile time loads regex data. Unfortunatley would require a restart for any pattern update
```

Comment:
This requires more than just restarting the application, wouldn't this require recompilation?

Perhaps something like:
"Load pattern data at compile time.  Recompiling the application is necessary after updating the pattern file."